scripts/lint.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+err=0
```

Comment:
Why not just `set -eu` as the first line of the script? Then any command that references undefined variables or that returns non-zero will crash immediately.

Reply:
oh nice i didnt know about that! thnx, making changes